components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.ts

```diff
@@ -26,21 +26,31 @@ export class ProjectsFilterDropdownComponent implements OnChanges {
   @Input() filterVisible = false;
 
   @Output() onSelection = new EventEmitter<ProjectsFilterOption[]>();
+  @Output() onOptionChange = new EventEmitter<ProjectsFilterOption[]>();
 
   editableOptions: ProjectsFilterOption[] = [];
+  filteredOptions: ProjectsFilterOption[] = [];
```

Comment:
Filtered options is merely a copy of the editable options so they can be filtered while maintaining the actual options.